integreat_cms/cms/models/regions/region.py

```diff
@@ -48,6 +49,17 @@
 logger = logging.getLogger(__name__)
 
 
+def validate_slug(slug):
+    """
+    Helper function to validate a slug is not a reserved slug
+
+    :param slug: The slug to be validated
+    :type slug: str
+    """
+    if slug in settings.RESERVED_REGION_SLUGS:
+        raise ValidationError(_("%(slug)s is a reserved slug."), params={"slug": slug})
```

Comment:
I would choose one consistent enforcement method, either validators or the clean method.

Reply:
I decided to go for the clean method